xkit.js

```diff
@@ -623,53 +617,8 @@ XKit = {
 		return to_return;
 	},
 	console: {
-		shown: false,
-		cache: "",
-		show: function() {
-			XKit.tools.add_css("#xkit_console { text-align: left; position: fixed; bottom: 0; left: 0; height: 110px; " +
-					" z-index: 1000000; width: 100%; background: black; color: #75fa96; font-family: Courier; " +
-					" padding: 10px; font-size: 10px; line-height: 14px; text-shadow: 0px 2px 3px black; " +
-					" overflow: scroll; background: rgba(0,0,0,0.53); }" +
-					" .xkit-toggle-extension-setting, .xkit-terminate-extension { " +
-					" text-decoration: underline; display: inline-block; " +
-					" cursor: pointer; margin-left: 15px; display: none; } ", "console");
-			$("body").append("<div id=\"xkit_console\">Welcome to XKit console!</div>");
-			$(document).on("click", ".xkit-terminate-extension", function() {
-				unload_extension($(this).attr('ext-id'));
-			});
-			$(document).on("click", ".xkit-toggle-extension-setting", function() {
-				var default_value = XKit.extensions[$(this).attr('ext-id')].preferences[$(this).attr('setting-id')].default;
-				var current_value = XKit.tools.get_extension_setting($(this).attr('ext-id'), $(this).attr('setting-id'), default_value);
-				XKit.console.add("current value = " + current_value);
-				if (current_value === true || current_value === "true") {
-					current_value = false;
-				} else {
-					if (current_value === false || current_value === "false") {
-						current_value = true;
-					}
-				}
-				XKit.tools.set_extension_setting($(this).attr('ext-id'), $(this).attr('setting-id'), current_value);
-				XKit.console.add("Default value = " + default_value + ", new value = " + current_value);
-				restart_extension($(this).attr('ext-id'));
-			});
-			if (XKit.console.cache !== "") {
-				$("#xkit_console").append("<br/>" + XKit.console.cache);
-				XKit.console.cache = "";
-			}
-		},
-		hide: function() {
-			XKit.tools.remove_css("console");
-			$("#xkit_console").remove();
-		},
 		add: function(text) {
-			if ($("#xkit_console").length > 0) {
-				$("#xkit_console").append("<br/>" + text);
-				var objDiv = document.getElementById("xkit_console");
-				objDiv.scrollTop = objDiv.scrollHeight;
-			} else {
-				XKit.console.cache = XKit.console.cache + ("<br/>" + text);
-			}
-			console.log(text);
+			console.warn("[DEPRECATED] XKit.console", text);
```

Comment:
Methods other than `log()` on console [are non-standard](https://developer.mozilla.org/en/docs/Web/API/console) and should not be used.